app/styles/atoms/progress-bar.less

```diff
@@ -0,0 +1,42 @@
+.oss-progress-bar--wrapper {
+  display: flex;
+  width: 100%;
+  justify-content: space-between;
+  align-items: center;
+  span {
+    font-weight: 200;
+    margin: 0 var(--spacing-px-6);
+  }
+}
+
+.oss-progress-bar {
+  width: 100%;
+  border-radius: var(--border-radius-sm);
+  background-color: #e5e5e5;
```

Comment:
no matching variable here ?

Reply:
Nope it wasn't in the dsm

Reply:
It might be something like Gray-100 or Gray-50

Reply:
Added  --color-gray-200